### PR TITLE
Couple of fixes / improvements

### PR DESCRIPTION
**Pull request by vespakoen**
Hey!

I added a `npm start` script that will compile the ES6 code to ES5, I removed an unused dependency and fixed the following warning:

```
Warning: You are manually calling a React.PropTypes validation function for the `value` prop on `BezierEditor`. This is deprecated and will not work in production with the next major version. You may be seeing this warning due to a third-party PropTypes library. See https://fb.me/react-warning-dont-call-proptypes for details.
```

Hope you like it =)

### CODE REVIEW

File: lib/BezierComponent.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+"use strict";
```
Comment (gre) [inline lib/BezierComponent.js:1]: please do not commit this file

Comment (gre) [reply lib/BezierComponent.js:1]: I mean lib/ folder.

**Review by gre — CHANGES_REQUESTED** [review]
yeah make sense, please remove lib/ from the committed files, i think it should probably be in .gitignore and explicitely added in package.json `files` file so lib get published, i can polish that latest.

thanks :)

**Comment by vespakoen** [comment]
Hmmz, the reason I added the lib folder is so I can import the module in my project.
I am using Browserify + Babel but by default, node_modules will not get "babelified".

Currently this module only makes ES6 files available through the NPM package, which is probably fine when you use something like Rollup, but not with most other packagers.

I think that lib should be included and "lib/BezierEditor.js" should even be the "main" in the package.json

**Comment by gre** [comment]
I agree for the problem you are raising and also that the the "main" should point to lib/, however, lib/ should not be versioned, 
here is an example:

https://github.com/gre/react-json2d/blob/master/package.json#L5-L8

**Comment by gre** [comment]
nowaday, I also tend to define a `"files"` field, that explicitely specify what should be pushed to NPM.
e.g.

```
"main": "lib/index.js",
  "files": [
    "src",
    "lib",
    "README.md"
  ],
```

**Comment by vespakoen** [comment]
Oh I see what you mean, sure, I guess you can pull it in and change it on your end right?

**Comment by gre** [comment]
(NB: prepublish script is implicitely called when typing `npm publish`)

**Comment by gre** [comment]
as you prefer, i can refactor that project conf tonight ;)

**Comment by gre** [comment]
Hi @vespakoen ,
A new version was released (0.6.0) using latest lib versions.
The npm packages is published without need to transform anything, lib/index.js is the entry, and also have the appropriate sourcemap files.

screen shot 2016-11-19 at 10 45 45

Example is also more up to date, using create-react-app bootstrap.

Cheers

**Pull request closed without merging**